lib/components/form/call-taker/mode-dropdown.tsx

```diff
@@ -23,21 +24,39 @@ type Props = {
  * rental companies).
  */
 function ModeDropdown({ modeDropdownOptions, onChangeModes }: Props) {
-  const [selectedMode, setSelectedMode] = useState(modeDropdownOptions[0].label)
+  const [selectedMode, setSelectedMode] = useQueryParam(
+    modeDropdownOptions[0].label
+  )
 
   const onChange = React.useCallback(
     (e: React.FocusEvent<HTMLSelectElement>) => {
       setSelectedMode(e.target.value)
-      const newModes = modeDropdownOptions.find(
-        (mdo) => mdo.label === e.target.value
-      )?.combination
-      onChangeModes(newModes || [])
     },
     []
   )
 
+  useEffect(() => {
+    if (selectedMode === undefined) {
```

Comment:
Could you do `!selectedMode` here? Or would that be different?

Reply:
`!selectedMode` is also true when `selectedMode` is `null`, `0`, and a few other javascript weirdnesses